connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -222,6 +223,14 @@ public void stop() {
         connectorStatusMetricsGroup.close();
 
         workerConfigTransformer.close();
+        executor.shutdown();
+        try {
+            if (!executor.awaitTermination(EXECUTOR_SHUTDOWN_TERMINATION_TIMEOUT_MS, TimeUnit.MILLISECONDS)) {
+                executor.shutdownNow();
+            }
+        } catch (InterruptedException e) {
+            executor.shutdownNow();
+        }
```

Comment:
Not sure if we should follow the 2-phase shutdown in "usage example" in java doc?
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/concurrent/ExecutorService.html

Reply:
I missed preserving interrupt status if `InterruptedException` occurs. Also added the log as suggested in java doc.